apps/core/lib/core/seed/payment_methods.ex

```diff
@@ -0,0 +1,45 @@
+defmodule Core.Seed.PaymentMethods do
+  @moduledoc """
+  Seeds supported PaymentMethods.
+
+  Snitch comes with two payment methods built-in:
+  1. check or cash
+  2. credit or debit cards
+
+  ## Check or Cash
+
+  This payment method is backed only by the `Core.Snitch.Data.Schema.Payments`
+  schema.
+
+  ## Cards
+
+  This payment method is backed by the `Core.Snitch.Data.Schema.CardPayments`
+  schema and table.
+
+  ## Roadmap
+
+  Snitch will support "Store Credits", which act like e-wallets for users.
+  """
+  use Core.Snitch.Data.Schema
+
+  def seed!() do
+    methods = [
```

Comment:
Since there is no schema or model for check/cash payments, IMO we should not seed this payment method in this PR.